src/main/java/woowacourse/shoppingcart/ui/CartItemController.java

```diff
@@ -7,20 +7,25 @@
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
+import woowacourse.auth.support.AuthenticationPrincipal;
 import woowacourse.shoppingcart.application.CartService;
-import woowacourse.shoppingcart.domain.Cart;
-import woowacourse.shoppingcart.ui.dto.ProductRequest;
+import woowacourse.shoppingcart.application.dto.CartItemResponse;
+import woowacourse.shoppingcart.application.dto.CartResponse;
+import woowacourse.shoppingcart.ui.dto.CartItemQuantityRequest;
+import woowacourse.shoppingcart.ui.dto.CartItemRequest;
+import woowacourse.shoppingcart.ui.dto.FindCustomerRequest;
 import woowacourse.shoppingcart.ui.dto.Request;
 
 @RestController
-@RequestMapping("/api/customers/{customerName}/carts")
+@RequestMapping("/api/customers/me/cart-items")
```

Comment:
저였다면 
```suggestion
@RequestMapping("/api/cart-items")
```
이렇게 했을수도 있을거 같아요.
어차피 이건 무조건 로그인한 사용자만 가능한 API이며, 지금과 같이 로그인한 사용자의 정보를 가져오는 부분은 모두 me/{resource}라고 해야 할 듯하지만, 단순히 리소스를 생각했을땐 위와 같은 느낌도 맞다고 생각이 드네요 :)

Reply:
ㅎㅎ 팀 논의 과정에서 말씀주신 path를 저도 제안했었는데요... "me" 라는 용어 자체가 "사용자가 요청한다" 라는 근시안적인 관점으로 바라보고 요청하는 것이기 때문에 곧바로 cart-items로 해도 괜찮지 않겠느냐 라고 의견을 제시했는데, 다른 팀원들의 의견은 cart-items이라는 것 자체가 customer의 소속이기 때문에 cart-items 자체는 의미가 불분명할 수 있다는 이야기를 주셨고, 저는 이에 동의해서 위와 같이 결정되었습니다.

스펙과 관련해서 정말 많은 논의가 오갔었는데, 무엇이 best-practice인지 감을 잡기가 너무 어렵네요 ㅎㅎ ㅠㅠ

Reply:
참 convention이기 때문에 더 그럴수 있다고 생각이 듭니다 :)
저도 그래서 제 의견이다. 라는 말로 남겨둔것이에요.
설계와 동일하다고 생각이 드는데요. 점점 경험해보다 보면 best-practice를 찾아갈 수 있지 않을까 합니다 :)